### DISCUSSION TITLE
Welcome to visit the "Generative AI for Beginners" course taught by AI.

### DISCUSSION THREAD

**Discussion by sailxjx** [microsoft/generative-ai-for-beginners#404] [Show and tell]
I've created a series of AI courses that allow for conversation with AI, using the content from Generative AI for Beginners (Thank you Korey and everyone who has contributed to this course).

What's even better is that in the courses involving text-to-image generation, you can directly try generating images through dialogue without the need to obtain OpenAI API access permissions.

The course is free, everyone is welcome to visit.

https://codekidz.ai/lesson-intro/generative-a-362093

**Comment by sailxjx** [comment]
And there is another course 'Artificial Intelligence for Beginners' https://codekidz.ai/lesson-intro/artificial-i-36275c

**Reply by Plug007** [reply]
Can we work together

**Reply by sailxjx** [reply]
@Plug007 How to? We need curriculum developer to create new course, if you are interested in contributing, please visit this page https://codekidz.ai/composer-program and contact me.

**Comment by Plug007** [comment]
Amazing
